testutil/sims/simulation_helpers.go

```diff
@@ -68,7 +68,7 @@ func SimulationOperations(app runtime.AppI, cdc codec.JSONCodec, config simtypes
 		}
 	}
 
-	simState.LegacyProposalContents = app.SimulationManager().GetProposalContents(simState) //nolint:staticcheck
+	simState.LegacyProposalContents = app.SimulationManager().GetProposalContents(simState) //nolint:staticcheck // TODO: remove once ProposalContents is removed
```

Comment:
can you remove this todo, I don't think it will be ever deleted, so it should be removed.